### DISCUSSION TITLE
Scrolling button

### DISCUSSION THREAD

**Discussion by MikelArdRi** [sezanzeb/input-remapper#585] [Q&A]
Is it possible to use the middle mouse button for scrolling (hold and move)? I use 3 button mouse without a scroll wheel

**Comment by jonasBoss** [comment]
sure, but only on the beta branch. You can record a input combination of holding the middle mouse button and moving the moues. and than map that to the REL_WHEEL axis.
see also: https://github.com/sezanzeb/input-remapper/blob/beta/readme/usage.md#analog-axis

**Comment by MikelArdRi** [comment]
where i can find beta binary for my fedora distribution?

пн, 26 дек. 2022 г. в 18:15, jonasBoss ***@***.***>:

> sure, but only on the beta branch. You can record a input combination of
> holding the middle mouse button and moving the moues. and than map that to
> the REL_WHEEL axis.
> see also:
> https://github.com/sezanzeb/input-remapper/blob/beta/readme/usage.md#analog-axis
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/sezanzeb/input-remapper/discussions/585#discussioncomment-4498223>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAHLZTF2R3P4LFCJXWZ2HVTWPGZA7ANCNFSM6AAAAAATJMDULI>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
С уважением, *Марат*.

**Reply by jonasBoss** [reply]
you probably need to install it from source. I don't think there are any packages for beta.

```
sudo pip install --no-binary :all: git+https://github.com/sezanzeb/input-remapper.git@beta
```
see also https://github.com/sezanzeb/input-remapper/tree/beta#manual
